pages/index.jsx

```diff
@@ -166,9 +183,9 @@ export default function Home() {
     onAfterPrint: () => console.log("printed"),
   });
 
-  const ifScaleUpOrDown = async () => {
+  const ifScaleUpOrDown = () => {
```

Comment:
here it needs to be async

Reply:
setScale is not an async function.
`'await' has no effect on the type of this expression.  `